DESCRIPTION

```diff
@@ -16,12 +18,27 @@ BugReports: https://github.com/inbo/inbospatial/issues
 Imports: 
     assertthat,
     httr,
+    magrittr,
```

Comment:
Why not use the base pipe `|>` instead of the magrittr pipe `%>%`? Development on the latter is stopped.

Reply:
I'm using it with curly braces, which the base pipe doesn't support:

https://github.com/inbo/inbospatial/blob/e2bc5155cec668fe0b470458cf48baaec8227c11/R/add_wms.R#L432-L445

But I might change it indeed and use the lambda notation to keep things short.

```r
setting <- TRUE
mtcars |> 
  (\(x) if (setting) x else subset(x, cyl == 4))() |> 
  nrow()
```

Although that still reads more akward IMHO because of the extra parentheses; compared to:

```r
library(magrittr)
mtcars %>%
  {if (setting) . else subset(., cyl == 4)} |> 
  nrow()
```